GDJS/Runtime/variable.js

```diff
@@ -114,6 +135,7 @@ gdjs.Variable.prototype.removeChild = function(childName) {
  * Remove all the children.
  *
  * If the variable is not a structure, nothing is done.
+ * @method
```

Comment:
I think that JSDoc does not need this annotation: https://4ian.github.io/GD-Documentation/GDJS%20Runtime%20Documentation/gdjs.Variable.html

And Typescript is also inferring automatically that these are methods so we can remove "@method" there :)

Reply:
Now that I took 10 minutes to add them, even if they are not mandatory, shouldn't we keep them? Or do you think it makes the whole thing unreadable?

Reply:
Not a big deal, so we can keep them. Though if they don't add anything to JSDoc or typescript, this mean it's noise and making the code longer for nothing so we should remove them.